reference/command/secure-channels.md

```diff
@@ -148,10 +148,10 @@ Let's create an end-to-end secure channel through an elastic relay in your Orche
 The [<mark style="color:blue;">Project</mark>](nodes.md#project) that was created when you ran `ockam enroll` offers an Elastic Relay Service. [<mark style="color:blue;">Delete any existing nodes</mark>](nodes.md#nodes) and then try this new example:
 
 ```
-» ockam project information --output json > project.json
+» ockam enroll
 
-» ockam node create a --project-path project.json
-» ockam node create b --project-path project.json
```

Comment:
The `--project-path` argument doesn't exist anymore and we now have the `ockam project import` command. But why do we want to add this extra step if this example is not a multi-machine example?

Reply:
Yes that seems unnecessary in that case.